src/events/guildMemberAdd.js

```diff
@@ -0,0 +1,10 @@
+const { Events } = require('discord.js');
+
+// Add a role when a new member joins
+module.exports = {
+	name: Events.GuildMemberAdd,
+	async execute(member) {
+		const welcomeRole = await member.guild.roles.fetch('1184218380509659226');
```

Comment:
```suggestion
		const welcomeRole = await member.guild.roles.fetch(env.WELCOME_ROLE_ID);
```
Plutôt utiliser une variable d'env au lieu d'une valeur magique.

Reply:
C'est tout bon, merci Cardiox12.